Source/WebService/Blackjack.WebService/ClientApp/src/app/types/player.ts

```diff
@@ -0,0 +1,15 @@
+import { Gamer } from "./gamer";
+
+export class Player extends Gamer {
+  /**
+   * The player's name
+   */
+  public name: string = '';
```

Comment:
Defaulting like this is not common.

Set this to

```suggestion
  public name: string;
```

Require a value via constructor if you need something here at instantiation time.

If you don't need something at instantiation then let whatever has an instance of this class set it whenever they have a value to provide.